docs/plugins/sonarqube.zh.md

```diff
@@ -0,0 +1,111 @@
+# sonarqube 插件
+
+`sonarqube` 插件用于部署、管理 [SonarQube](https://www.sonarqube.org/) 应用。
+
+## 1. 前置要求
+
+- 有一个可用的 Kubernetes 集群，版本 1.19+。
+- 运行 Sonarqube 小规模服务至少需要 2GB 的 RAM 内存。
+- Sonarqube 官方镜像目前只支持 linux/amd64 架构。
+- 更多硬件配置可参考[官方网站](?https://docs.sonarqube.org/latest/requirements/hardware-recommendations/)。
+
+## 2、部署架构
+
+Sonarqube 内部会使用 Elastcisearch 来做搜索的索引，所以生产环境中需要通过挂载目录的方式持久化数据。
+另外 Sonarqube 也需要一个外部数据库来存储数据，目前支持 `Postgresql`，`Oracle`，`Microsoft SQL Sever`，默认使用 `Postgresql`。
+
+## 3、开始部署
+
+下文将介绍如何配置 `sonarqube` 插件，完成 Sonarqube 应用的部署。
+
+### 3.1、默认配置
+
+`sonarqube` 插件的配置项多数都有默认值，具体默认值信息如下表：
+
+| 配置项              | 默认值                    | 描述                                 |
+| ----               | ----                     | ----                                |
+| chart.chart_name   | sonarqube/sonarqube      | helm chart 包名称                    |
```

Comment:
new config item: `chartPath` and some configs has changed. https://github.com/devstream-io/devstream/pull/1088

Reply:
it's still unresolved